### PR TITLE
Doc changes

### PR DESCRIPTION
**Pull request by coreydotnet**
Please review general approach for docs.

Feel free to leave comments for dynamic router content but it has a lot more work to go.

### CODE REVIEW

File: projects/showcase/src/app/dynamic-router/dynamic-router.component.html [hunk 1]
```diff
@@ -1,3 +1,175 @@
...
+&lt;ion-item [button]="true" [iptDynamicRouterLink]="NavigationTargets.Tasks"&gt;
+    &lt;ion-label&gt;Tasks&lt;/ion-label&gt;
+&lt;/ion-item&gt;</pre>
+                </ion-card-content>
+            </ion-card>            
+        </li>
+    </ol>
+
+    <h3 id="nav-from-ts">Navigation From TypeScript Components <a routerLink="." fragment="nav-from-ts">#</a></h3>
+    <form (ngSubmit)="goToTarget()">
```
Comment (coreydotnet) [inline projects/showcase/src/app/dynamic-router/dynamic-router.component.html:108]: This was supposed to be a demo where the user can pick a target and click a button to navigate but it doesn't work. The `goToTarget()` method doesn't get hit and it instead navigates to `http://localhost:4200/dynamic-router?ion-sel-0=1`

Comment (bradley-dotnet) [reply projects/showcase/src/app/dynamic-router/dynamic-router.component.html:108]: Need to import `FormsModule` as well for this to work; ngSubmit is not doing anything right now

File: projects/showcase/src/app/dynamic-router/dynamic-router.component.ts [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
+    styleUrls: ['./dynamic-router.component.scss']
 })
 export class DynamicRouterComponent {
 
-  constructor(
-    private readonly appStore: AppStateStoreService
-  ) { }
+    constructor(
+        private readonly appStore: AppStateStoreService,
+        private dynamicRouter: DynamicRouter<NavigationTargets>
```
Comment (bradley-dotnet) [inline projects/showcase/src/app/dynamic-router/dynamic-router.component.ts:16]: Should be consistent with use of readonly

File: projects/showcase/src/app/dynamic-router/dynamic-router.component.ts [hunk 3]
```diff
@@ -1,19 +1,34 @@
...
+    public JSON = JSON;
+
+    public selectedTarget = new FormControl(NavigationTargets.Dialog);
+
+    public ionViewWillEnter(): void {
+        this.appStore.setCurrentPage('Dynamic Router');
+    }
+
+    public goToTarget(): void {
+        console.log(this.selectedTarget.value);
```
Comment (bradley-dotnet) [inline projects/showcase/src/app/dynamic-router/dynamic-router.component.ts:30]: Generally don't want to check in console.log

File: projects/showcase/src/styles.scss [hunk 4]
```diff
@@ -24,3 +24,41 @@
...
+        margin-bottom: 2rem;
+    }
+
+    h1 a,
+    h2 a,
+    h3 a,
+    h4 a,
+    h5 a,
+    h6 a {
+        display: none;
```
Comment (bradley-dotnet) [inline projects/showcase/src/styles.scss:52]: Why are we hiding all links in headers? At least leave a comment here

**Review by bradley-dotnet — COMMENTED** [review]
Mostly good; would be nice to cleanup unused functionality. Also fix the form bug

**Review by bradley-dotnet — APPROVED** [review]

**Pull request merged**